control-plane/agents/src/bin/core/main.rs

```diff
@@ -152,12 +160,17 @@ async fn main() -> anyhow::Result<()> {
     let cli_args = CliArgs::args();
     utils::print_package_info!();
     println!("Using options: {cli_args:?}");
-    utils::tracing_telemetry::init_tracing_with_eventing(
-        "agent-core",
-        cli_args.tracing_tags.clone(),
-        cli_args.jaeger.clone(),
-        cli_args.events_url.clone(),
-    );
+    let fmt_style = cli_args.fmt_style.clone();
+    utils::tracing_telemetry::TracingTelemetry::builder()
+        .with_writer(FmtLayer::Stdout)
+        .with_style(fmt_style)
```

Comment:
Add Copy to `FmtStyle`, so we can pass it directly:

```suggestion
    utils::tracing_telemetry::TracingTelemetry::builder()
        .with_writer(FmtLayer::Stdout)
        .with_style(cli_args.fmt_style)
```